docs/source/form_boundary_conditions.rst

```diff
@@ -0,0 +1,191 @@
+.. default-role:: math
```

Comment:
Don't make a whole copy of the Dirichlet BC documentation. Just add to the existing BC documentation.